types/amqplib/index.d.ts

```diff
@@ -71,6 +72,12 @@ export interface ConfirmChannel extends Channel {
 }
 
 export const credentials: {
+    amqplain(username: string, password: string): {
+        mechanism: string;
+        response(): Buffer;
+        username: string;
+        password: string;
+    };
```

Comment:
This seems to be identical to the `plain` function that is on line 85. What is the difference?

Reply:
Ah, nevermind - I see this here: https://github.com/squaremo/amqp.node/blob/main/lib/credentials.js#L25